tools/autoconf/Makefile

```diff
@@ -7,18 +7,20 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=autoconf
-PKG_VERSION:=2.69
+PKG_VERSION:=2.71
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=@GNU/autoconf
-PKG_HASH:=64ebcec9f8ac5b2487125a86a7760d2591ac9e1d3dbd59489633f9de62a57684
+PKG_HASH:=f14c83cfebcc9427f2c3cea7258bd90df972d92eb26752da4ddad81c87a0faa4
 
 include $(INCLUDE_DIR)/host-build.mk
 
 HOST_CONFIGURE_VARS += \
+	EMACS="no" \
 	PERL="/usr/bin/env perl"
 
 define Host/Compile
+	-$(call Host/Compile/Default,--touch install-man1) # disable building of man pages
 	$(call Host/Compile/Default)
```

Comment:
Why do you call `Host/Compile/Default` twice?

Reply:
see first comment by me. CI build fails to build manual, so we just skip it by touching the files